docs/source/writing.rst

```diff
@@ -61,6 +61,10 @@ Create the package with cookiecutter
 
 #. Use cookiecutter to create a new suitcase package. Just follow the prompts.
 
+   .. note:: `suitcase-cookiecutter <https://github.com/bluesky/suitcase-
```

Comment:
I believe a link cannot have a line break in it. I make a habit of building the documentation locally to check that it renders as expected.

Reply:
Actually, I did. and looks a line break there is fine.

Reply:
Cool. I don’t think that used to work, good to know.